packages/react-components/src/organisms/TeamCreateOutputForm.tsx

```diff
@@ -160,9 +161,10 @@ const TeamCreateOutputForm: React.FC<TeamCreateOutputFormProps> = ({
         authors.length !== 0 ||
         identifierType !== ResearchOutputIdentifierType.None ||
         identifier !== '' ||
+        labCatalogNumber !== '' ||
         teams.length !== 1 // Original team
       }
-      onSave={() => {
+      onSave={async () => {
```

Comment:
This isn't async, Do we want to remove this?

Reply:
done